usr.bin/minimal_count_stats/minimal_count_stats.c

```diff
@@ -28,65 +31,113 @@
  * OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
  * SUCH DAMAGE.
  */
-
 #include <sys/types.h>
 #include <sys/sysctl.h>
+#include <sys/time.h>
 #include <sys/wait.h>
 
+#include <assert.h>
 #include <err.h>
 #include <getopt.h>
 #include <inttypes.h>
 #include <libgen.h>
 #include <spawn.h>
 #include <statcounters.h>
 #include <stdbool.h>
+#include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include <stdio.h>
-#include <sysexits.h>
+#include <time.h>
 
 extern char **environ;
 
+// clang-format off
```

Comment:
Not style(9)

Reply:
The file already uses // comments, and since this is a utility that's highly unlikely to be upstreamed I don't think this needs to be fixed.